rules/autoconsent/amazon.json

```diff
@@ -1,8 +1,9 @@
 {
     "name": "amazon.com",
+    "vendorUrl": "https://amazon.com/",
     "prehideSelectors": ["span[data-action=\"sp-cc\"][data-sp-cc*=\"rejectAllAction\"]"],
     "detectCmp": [{ "exists": "span[data-action=\"sp-cc\"][data-sp-cc*=\"rejectAllAction\"]" }],
     "detectPopup": [{ "visible": "span[data-action=\"sp-cc\"][data-sp-cc*=\"rejectAllAction\"]" }],
-    "optIn": [{ "waitForVisible": "#sp-cc-accept" }, { "wait": 500 }, { "click": "#sp-cc-accept" }],
-    "optOut": [{ "waitForVisible": "#sp-cc-rejectall-link" }, { "wait": 500 }, { "click": "#sp-cc-rejectall-link" }]
+    "optIn": [{ "waitForVisible": "#sp-cc-accept" }, { "wait": 5000 }, { "click": "#sp-cc-accept" }],
+    "optOut": [{ "waitForVisible": "#sp-cc-rejectall-link" }, { "wait": 5000 }, { "click": "#sp-cc-rejectall-link" }]
```

Comment:
I tried to find a better way, but the layout is not really giving any hints. The delay is not noticeable because the popup is pre-hidden all this time